### DISCUSSION TITLE
Netplay not working after copying install to a second sd card/mm+

### DISCUSSION THREAD

**Discussion by Madeup1456** [OnionUI/Onion#1608] [Q&A]
I'm wondering if there any unique id assigned at install or some reason I couldn't use netplay, I've tried easy netplay and using the same Wi-Fi network.  Tried snes, and Genesis.  Using easy netplay  It makes it to, trying to join the hotspot, then waiting for an IP, falls back on the static IP and eventually fails.

If it's worth just doing a fresh install I'm willing, didn't know if it might be related.  Ones a v4 and the other is at least a v3 or older if that matters.

**Comment by roo1234** [comment]
Strange. I have 2 with the same onion image 64G and they net play perfectly, using the menu (start) for host and client. Picodrive.
Couldn't make it connect with another OS though, it connects and disconnects immediately.
Have to add they are mm+
